Terminal.Gui/Core/View.cs

```diff
@@ -2334,8 +2414,22 @@ void CollectAll (View from, ref HashSet<View> nNodes, ref HashSet<(View, View)>
 				if (autoSize != v) {
 					autoSize = v;
 					TextFormatter.NeedsFormat = true;
-					SetNeedsLayout ();
-					SetNeedsDisplay ();
+					UpdateTextFormatterText ();
+					ProcessResizeView ();
+				}
+			}
+		}
+
+		/// <summary>
+		/// Get or sets a flag if the wrapped text will keep the trailing spaces or
```

Comment:
See my other comment where I rewrote the text fo this. (Shouldn't `<inheritdoc/>` be used?)

Reply:
> See my other comment where I rewrote the text fo this. (Shouldn't `<inheritdoc/>` be used?)

I don't agree because the `Responder` class doesn't inherits the `TextFormatter` class. The user would like to see a documentation pointing to the `TextFormatter.Text` and the `TextFormatter.WordWrap`. It will be like this:


```cs
		/// <summary>
		/// Gets or sets a flag that determines whether <see cref="TextFormatter.Text"/> will have trailing spaces preserved
		/// or not when <see cref="TextFormatter.WordWrap"/> is enabled. If `true` any trailing spaces will be trimmed when
		/// either the <see cref="Text"/> property is changed or when <see cref="TextFormatter.WordWrap"/> is set to `true`.
		/// The default is `false`.
		/// </summary>
```